tests/Get-DbaDBFileGroup.Tests.ps1

```diff
@@ -0,0 +1,66 @@
+$CommandName = $MyInvocation.MyCommand.Name.Replace(".Tests.ps1", "")
+Write-Host -Object "Running $PSCommandPath" -ForegroundColor Cyan
+. "$PSScriptRoot\constants.ps1"
+
+Describe "$CommandName Unit Tests" -Tag 'UnitTests' {
+    Context "Validate parameters" {
+        [object[]]$params = (Get-Command $CommandName).Parameters.Keys | Where-Object {$_ -notin ('whatif', 'confirm')}
+        [object[]]$knownParameters = 'SqlInstance', 'SqlCredential', 'Database', 'InputObject', 'FileGroup', 'EnableException'
+        $knownParameters += [System.Management.Automation.PSCmdlet]::CommonParameters
+        It "Should only contain our specific parameters" {
+            (@(Compare-Object -ReferenceObject ($knownParameters | Where-Object {$_}) -DifferenceObject $params).Count ) | Should Be 0
+        }
+    }
+}
+<#
+    Integration test should appear below and are custom to the command you are writing.
+    Read https://github.com/sqlcollaborative/dbatools/blob/development/contributing.md#tests
+    for more guidence.
+#>
+Describe "$commandname Integration Tests" -Tags "IntegrationTests" {
+    BeforeAll {
+        $random = Get-Random
+        $multifgdb = "dbatoolsci_multifgdb$random"
+        Remove-DbaDatabase -Confirm:$false -SqlInstance $script:instance1 -Database $multifgdb
```

Comment:
Please use `instance2` for individual commands. The [group configuration](https://github.com/sqlcollaborative/dbatools/blob/prerelease/tests/pester.groups.ps1) you will see instance1 is a 2008 R2 server, unless you see a need for this to be run against that version?